jre_emul/android/platform/libcore/luni/src/test/java/libcore/java/nio/file/attribute/UserPrincipalNotFoundExceptionTest.java

```diff
@@ -17,13 +17,11 @@
 package libcore.java.nio.file.attribute;
 
 import org.junit.Test;
-import junit.framework.TestCase;
-
 import static org.junit.Assert.assertEquals;
 
 import java.nio.file.attribute.UserPrincipalNotFoundException;
 
-public class UserPrincipalNotFoundExceptionTest extends TestCase {
+public class UserPrincipalNotFoundExceptionTest {
```

Comment:
Why the change? Were JUnit4 tests not being run because the runner categorized this as a JUnit3 test class?

Reply:
The previous version was added by mistake in libcore.java.nio.channels.attribute. Had to delete it and add it back in the correct place. The old file in the incorrect location was not being found by JUnit unless `extends TestCase` was added. Now that the file is in the correct folder there is no need for the extend and file can be identical to libcore version